manifests/config.pp

```diff
@@ -1,7 +1,7 @@
 # == Class icingaweb2::config
 #
-class icingaweb2::config {
-  assert_private()
```

Comment:
Please update your stdlib and revert this line. ;)

https://github.com/puppetlabs/puppetlabs-stdlib/commit/bf8e5b0573f78f2c2c780494a735bfac25d4488d